x-pack/test/ingest_manager_api_integration/apis/fixtures/package_registry_config.yml

```diff
@@ -1,3 +1,3 @@
   package_paths:
-    - /registry/packages/package-storage
-    - /registry/packages/test-packages
\ No newline at end of file
+    - /packages/production
```

Comment:
These changes will go away once this PR is merged: https://github.com/elastic/kibana/pull/70716